content/de/cloud-native-security.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cloud Native Security
+status: Completed
+category: Konzept
+---
+
+## Was es ist
+
+Cloud Native Security beschreibt den Ansatz, Konzepte aus dem Bereich Cyber Security in [Cloud Native Anwendungen](/cloud-native-apps/) zu integrieren. 
+Dabei ist vorgesehen, dass Sicherheitsbetrachtungen Teil des gesamten Software-entwicklungs und -Lebenszyklus ist, von der Entwicklung bis zur Inbetriebnahme und Wartung.  
+Cloud Native Security versucht, die gleichen Standards wie herkömmliche Sicherheitsmodelle zu gewährleisten,
+ist dabei allerdings angepasst an die Besonderheiten von Cloud Native Umgebungen. 
+Eine hohe Frequenz von Änderungen des Codes und eine sehr volatile Infrastruktur sind Bespiele für die Eigenschaften von Cloud Native Umgebungen. 
+Cloud Native Security ist eng mit dem Methodologie des [DevSecOps](/devsecops/) verbunden.
+
+## Welches Problem es löst
+
+Traditionelle Sicherheitsmodelle basieren auf einer Reihe von Annahmen, die nicht mehr länger gültig sind. 
+Cloud Native Anwendungen ändern sich häufig, verwenden eine große Anzahl von Open-Source-Tools und -Bibliotheken, 
+werden in der Regel in einer Infrastruktur betrieben, deren Änderungen sich der Kontrolle von Entwicklern entziehen. 
```

Comment:
der Satz entspricht auch sinngemäß mMn nicht dem englischen Original:

Dort heißt es eher: 
Cloud Native Anwendungen ändern sich häufig, verwenden eine große Anzahl von Open-Source-Tools und -Bibliotheken, werden oft in einer vom Anbieter kontrollierten Infrastruktur ausgeführt und unterliegen schnellen Infrastrukturänderungen.